homeassistant/components/tuya/switch.py

```diff
@@ -88,6 +88,14 @@
             name="Switch",
         ),
     ),
+    # Irrigator
+    # https://developer.tuya.com/en/docs/iot/categoryggq?id=Kaiuz1qib7z0k
+    "ggq": (
+        SwitchEntityDescription(
+            key=DPCode.SWITCH,
```

Comment:
It could be start as well?

Reply:
Yes

Reply:
In that case, we need to support that, as right now, it only handles the switch

Reply:
As mine is also not supporting start I need to add a virtual device as proposed by you and test it.